rs/rosetta-api/icp/tests/system_tests/test_cases/neuron_management.rs

```diff
@@ -163,3 +164,149 @@ fn test_set_neuron_dissolve_delay_timestamp() {
+                .collect(),
+            )
+            .with_governance_canister()
+            .build()
+            .await;
+
+        // Stake the minimum amount 100 million e8s
+        let staked_amount = 100_000_000u64;
+        let neuron_index = 0;
+        let from_subaccount = [0; 32];
+
+        env.rosetta_client
+            .create_neuron(
+                env.network_identifier.clone(),
+                &(*TEST_IDENTITY).clone(),
+                RosettaCreateNeuronArgs::builder(staked_amount.into())
+                    .with_from_subaccount(from_subaccount)
+                    .with_neuron_index(neuron_index)
+                    .build(),
+            )
+            .await
+            .unwrap();
+
+        // See if the neuron was created successfully
+        let agent = get_test_agent(env.pocket_ic.url().unwrap().port().unwrap()).await;
+        let neuron = list_neurons(&agent).await.full_neurons[0].to_owned();
+        let dissolve_delay_timestamp = match neuron.dissolve_state.unwrap() {
+            // When a neuron is created its dissolve delay timestamp is set to now which corresponds to the state DISSOLVED
+            DissolveState::WhenDissolvedTimestampSeconds(dissolve_delay_timestamp) => {
+                dissolve_delay_timestamp
+            }
+            k => panic!(
+                "Neuron should be in WhenDissolvedTimestampSeconds state, but is instead: {:?}",
+                k
+            ),
+        };
+        let start_dissolving_response = TransactionOperationResults::try_from(
+            env.rosetta_client
+                .start_dissolving_neuron(
+                    env.network_identifier.clone(),
+                    &(*TEST_IDENTITY).clone(),
+                    neuron_index,
+                )
+                .await
+                .unwrap()
+                .metadata,
+        )
+        .unwrap();
+        // If the neuron is already in dissolving state, the response the function to dissolve the neuron will succed but it will not change the state of the neuron
+        assert_eq!(
+            start_dissolving_response.operations.first().unwrap().status,
+            Some("COMPLETED".to_owned())
+        );
+        let neuron = list_neurons(&agent).await.full_neurons[0].to_owned();
+        match neuron.dissolve_state.unwrap() {
+            DissolveState::WhenDissolvedTimestampSeconds(d) => {
+                assert!(dissolve_delay_timestamp >= d);
+            }
+            k => panic!(
+                "Neuron should be in DissolveDelaySeconds state, but is instead: {:?}",
+                k
+            ),
+        };
+
+        let additional_dissolve_delay = 1000;
+        let new_dissolve_delay_timestamp = SystemTime::now()
+            .duration_since(UNIX_EPOCH)
+            .unwrap()
+            .as_secs()
+            + additional_dissolve_delay;
+
+        // To be able to set the dissolve delay timestamp we need to set the state machine to live again
```

Comment:
Is this comment a copy&paste leftover? I don't see an explicit call to start or stop the live mode, and the checks don't involve timestamps so I could imagine this is not necessary in this test?

Reply:
Yes, I changed it